css-transitions-2/Overview.bs

```diff
@@ -120,6 +122,42 @@ the reversing shortening factor.
 The 'transition-delay' property specifies the [=start delay=]
 of the transition's associated [=animation effect=].
 
+## The 'transition-behavior' Property ## {#transition-behavior-property}
+
+The 'transition-behavior' property specifies whether transitions will be
+started or not for discrete properties.
+
+<pre class="propdef">
+  Name: transition-behavior
+  Value: normal | allow-discrete
+  Initial: normal
+  Applies to: All elements
+  Inherited: no
+  Percentages: N/A
+  Computed value: as specified
+  Canonical order: per grammar
+  Animation type: not animatable
+</pre>
+
+The syntax for specifying 'transition-behavior' is as follows:
+
+<div class="prod">
+  <dfn type id="transition-behavior-value">&lt;transition-behavior-value&gt;</dfn> = [ 'normal' | 'allow-discrete' ]
```

Comment:
This could be a bit simpler, see the source of css-backgrounds-3 for an example.  Though I admit it matches the complexity in css-transitions-1...